test/hotspot/jtreg/TEST.groups

```diff
@@ -444,6 +444,19 @@ tier1 = \
   :tier1_runtime \
   :tier1_serviceability
 
+tier2 = \
+  -:tier1 \
```

Comment:
Not certain but I think the excludes have to come last. Also in theory shouldn't be needed because the tier2_x group should already exclude the tier1_x group.

Reply:
Right. Actually now I am looking at it, these exclusions are not required. The included tierX* groups should exclude the tests from other tiers. See new commit.